src/plot_utils.py

```diff
@@ -79,7 +79,7 @@ def mi_hist(bdb, generator, col1, col2, num_samples=1000, bins=5):
 
 
 def heatmap(bdb, bql, vmin=None, vmax=None, row_ordering=None,
-        col_ordering=None):
+            col_ordering=None):
```

Comment:
I appreciate other instances of wrong-continued-indentation though... I'd rather not disable them all.